components/Workout/Workout.js

```diff
@@ -67,13 +72,14 @@ export default function Workout({ workout, handleDelete, handleEditWorkout }) {
             </ModalContent>
           </ModalOverlay>
         ) : null}
-
-        <StyledEditDeleteWrapper>
-          <StyledButtonDelete onClick={toggleDeleteMode}>
-            Delete
-          </StyledButtonDelete>
-          <StyledButtonEdit onClick={toggleEditMode}>Edit</StyledButtonEdit>
-        </StyledEditDeleteWrapper>
+        {spotlightMode ? null : (
```

Comment:
```suggestion
        {!spotlightMode && (
```
Honestly, I find it strange when the code behind the ? in the ternary is null. I understand that you only want something to be displayed if the condition is not true. So why don't you use the NOT operator `!` and `&&`? I personally find that much easier to read, but it doesn't change your logic. 🤷‍♀️